### PR TITLE
Measure 16: CDF-HH

### PR DESCRIPTION
**Pull request by jdinh8124**
## Purpose
Complete Measure 16: CDF-HH

#### Linked Issues to Close
https://qmacbis.atlassian.net/jira/software/c/projects/OY2/boards/208?modal=detail&selectedIssue=OY2-8978

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [ ] This PR has an associated issue or issues.
- [ ] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

**Review by daniel-belcher — APPROVED** [review]
Matches Figma as far as I could tell and fills ticket reqs, looks good to me

File: services/ui-src/src/measures/2021/CDFHH/index.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+          )}
+          <CMQ.CombinedRates healthHomeMeasure />
+          {showOptionalMeasureStrat && (
+            <CMQ.OptionalMeasureStrat
+              categories={PMD.categories}
+              qualifiers={PMD.qualifiers}
+              rateMultiplicationValue={100}
+              performanceMeasureArray={performanceMeasureArray}
+              adultMeasure={false}
+              allowNumeratorGreaterThanDenominator
```
Comment (daniel-belcher) [inline services/ui-src/src/measures/2021/CDFHH/index.tsx:67]: Just noticed this, is the `allowNumeratorGreaterThanDenominator` intentional?

Comment (abelzer212) [reply services/ui-src/src/measures/2021/CDFHH/index.tsx:67]: Yeah, I think that's only for the PQ measures and should be removed

Comment (abelzer212) [inline services/ui-src/src/measures/2021/CDFHH/index.tsx:67]: This needs to be removed

**Review by abelzer212 — CHANGES_REQUESTED** [review]

**Review by abelzer212 — DISMISSED** [review]

**Pull request merged**